### PR TITLE
Minor fixes to improve code legibility / smooth interaction with Classic

### PR DESCRIPTION
**Pull request by MikeStower**

### CODE REVIEW

File: Scripts/WheatManagerC3.cs [hunk 1]
```diff
@@ -0,0 +1,184 @@
+using System;
```
Comment (BrianCollinss) [inline Scripts/WheatManagerC3.cs:1]: Did it work, @MStowerUQ?

Comment (MikeStower) [reply Scripts/WheatManagerC3.cs:1]: The scripts are producing output, but I think there is some more fine tuning required before merging - the numbers still aren't quite right. Having a discussion with Alex about it, shouldn't take too long to resolve.

**Comment by BrianCollinss** [comment]
How are the scripts doing now?

**Comment by MikeStower** [comment]
> How are the scripts doing now?

Looking good.

I think one or two of the parameters in C3 might accidentally be using the CCM value, but otherwise they seem to be working. Alex has been running simulations with them and getting results he's happy with. I'm currently in the process of updating my own tests with the new data.

**Comment by docalexwu** [comment]
Yes.
I made sure all of the values in all three scripts in our dropbox folder are the correct set.

**Comment by BrianCollinss** [comment]
Are we close to what you had with the old code?

**Comment by docalexwu** [comment]
Same to the many decimal places. We've not updated any of the science.
I think the issue was more about getting a working DCaPST version again as we've been making lots of changes.

**Comment by BrianCollinss** [comment]
Great! So we have a finalised DCaPST version and tested scripts now.

**Comment by docalexwu** [comment]
I think we are pretty close.
Mike just has to make DCaPST work with Maize Classic and I will check it.

**Pull request merged**